docs/docs/glossary.md

```diff
@@ -0,0 +1,112 @@
+
+[[toc]]
+
+## General
+
+### Access Token
+This is a general term referring to a string issued by an identifying service like an [identity provider] that validates the holder to have a specific set of permissions. Most of the access tokens discussed in our docs are [JWTs].
+
+### Identity Provider
+An identity provider (**IdP**) is used to [authenticate] a user, i.e. confirm their identity. Pomerium uses external IdPs to better integrate into existing environments, and to achieve strong separation of services.
+
+### Identity-aware Proxy
+A [proxy](https://en.wikipedia.org/wiki/Proxy_server) is an intermediate service between a one or more clients or servers. Most of the proxies discussed in our docs are technically [reverse proxies](https://en.wikipedia.org/wiki/Reverse_proxy), sitting between one or more severs and all clients, providing a single point of ingress into a system.
+
+An identity-aware proxy can provide contextual access to specific services based on the identity of the client. In Pomerium's case, identity is provided by the client in the form of a [JWT] issued by the [identity provider].
+
+### JavaScript Object Notation
+Commonly shortened to **JSON**, [JavaScript object notation](https://en.wikipedia.org/wiki/JSON) is a common format used to represent and share structured sets of data as arrays of key-value pairs.
+
+### JSON Web Key Sets
+Usually abbreviate as **JWKS**, this is a [JSON]-formatted set of one or more keys provided by a trusted issuer and used by service to verify [JWTs] provided by a client.
+
+### JSON Web Token
+Often referred to as **JWTs**, a JSON web token is a [JSON]-formatted string provided to a user by an [identity provider], which validates the user's identity to subsequent services (such as an [identity-aware-proxy]).
+
+### Namespace
+"Namespaces" is an over-saturated term, having different meanings in different contexts. [Pomerium Enterprise][pom-namespace] uses Namespaces to provide separation of access and control to [routes]. Kubernetes uses their [namespaces][k8s-namespace] to isolate groups of resources within a cluster.
+
+### Perimeter
+The term "Perimeter" in the context of Pomerium and general networking usually refers to your internal network, and common tools like firewalls used to restrict access to it. Historically, most security models used the perimeter as the main layer of protection to a network system. The principles of [zero trust] assume that the perimeter can be (and likely is) compromised, and require security between each connection, including those between internal services.
+
+### Policy
+Pomerium allows administrators to define policies dictating what combination of users, groups, devices, etc, have access to protected services. Open-source Pomerium defines a unique policy to every [route], while Pomerium Enterprise can define reusable policies at the global and [namespace] level.
```

Comment:
```suggestion
Pomerium allows administrators to define authorization policies dictating what combination of users, groups, devices, etc, have access to protected services. Open-source Pomerium defines a unique policy to every [route], while Pomerium Enterprise can define reusable policies at the global and [namespace] level.
```

Reply:
I think we should mention that we support expressing authorization policy in a way that enables folks to implement RBAC, ABAC, least user privilege and any other form of access control they ale looking to accomplish with links to each Wikipedia.